homeassistant/components/rmvtransport/sensor.py

```diff
@@ -104,6 +72,31 @@
     async_add_entities(sensors)
 
 
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up a RMV departure sensor based on a config entry."""
+    timeout = entry.data.get(CONF_TIMEOUT, DEFAULT_TIMEOUT)
+
+    session = async_get_clientsession(hass)
```

Comment:
We shouldn't need the session here. We already have set up the data client in the component.